data/models.dvc

```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: b5eec55168aac3c37ebb280278aba1d3.dir
-  size: 39869012
-  nfiles: 28
+- md5: 5a2212cd3f931d028a95ec55b6ec2f7b.dir
+  size: 30209284
+  nfiles: 22
```

Comment:
Added changed `models.dvc` - is this nfile change correct? Were models removed from the dvc repository recently?

Reply:
`models.dvc` inside the `master` branch is 28 so I think this branch is the only one where the change is occurring. I recommend pulling the two model files to a separate location and doing a `dvc pull` using the prior `models.dvc` file.